### DISCUSSION TITLE
Feature Request: Export "Group Metrics" from Hunt

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#5507] [Ideas]
The hunt feature is nice, and I use it daily. However, when I find something that needs to be acted on, I have to export evidence. To do this, I have to take my query from hunt and put it in Kibana, and then export a csv file from there. I would be nice if there were an export feature in Hunt to be able to export straight from the Hunt interface instead of switching to Kibana.

**Comment by dougburks** [comment]
We'd like to add features like this, but we already have quite a backlog. If you know of any experienced Golang developers, please point them to https://blog.securityonion.net/2021/08/security-onion-solutions-is-looking-for.html. Thanks!
